### ISSUE TITLE
[1.16.1] Messed up textures for Sugar Cane

### ISSUE THREAD

**Issue by HauntedSniper** [CyanideX/Unity-Extras#6]
**Describe the Issue**
The textures of the sugar cane "block" are not working properly and look weird.

**To Reproduce**
1. Place sugar cane or find sugar cane in the world.
2. See error.

**Expected Behavior**
The textures should look normal and working properly

**Screenshots**
![image](https://user-images.githubusercontent.com/67271557/100950418-e46ba080-34d1-11eb-8482-7f5e7554ac2d.png)

**Additional Context**
I saw people having a problem with this in the curseforge comments and you asked for someone to make an issue report here but no one did. I am having the same problem and I might as well report it here.

**Comment by ChloeDawn** [comment]
Which version of Unity are you using? Are you also using any additional Unity packs?

**Comment by HauntedSniper** [comment]
It's the latest version of Unity and Unity Extras available in the curseforge page for 1.16, other people have noticed the same problem and posted it on the comments of Unity Extras.

**Comment by ChloeDawn** [comment]
Found the problem! I had fixed this issue in July (https://github.com/Unity-Resource-Pack/Unity-Extras/commit/88911f60bf52d72def206ee47046d1f2ac5526bd) but on CurseForge there has not been a release for this fix. As a temporary fix, until we can get a new release on CurseForge out, I recommend [downloading this repository as a ZIP](https://github.com/Unity-Resource-Pack/Unity-Extras/archive/1.16.zip), extracting it (it downloads as a folder within the ZIP IIRC) and then using that as a resource pack.

EDIT: Nope! Other way around! We had changed the model in Unity and updated in Extras to account for it, but only Extras got a CurseForge release! Do the same but with the main Unity repo.

This is how it should look with the current changes. Reverted to the original cross model with varies heights on the top.

![Screenshot of sugar cane](https://i.imgur.com/Fqljy7M.png)
